src/Blog/BlogPlaceholder.scss

```diff
@@ -0,0 +1,123 @@
+@import "../styles/typography";
+.blog-section {
```

Comment:
This component seems to get weird when screen size is under 550px. I can't really tell why its doing this but you should investigate and fix

Reply:
Could you send me a screenshot? Have a feeling it could be the navbar pushing it out but unsure